main.cpp

```diff
@@ -108,15 +108,38 @@ struct CarWash
     You'll need to insert the Person struct from the video in the space below.
  */
 
+struct LeftFoot 
+{
+    int step;
+};
+struct RightFoot
+{ 
+    int step;
+};
+
+struct Person 
+{
+    
+    LeftFoot leftFoot;
+    RightFoot rightFoot;
 
+    void stepForward(LeftFoot leftFoot, RightFoot rightFoot);
 
+    int stepSize (LeftFoot leftFoot, RightFoot rightFoot);
 
+    void run();
+};
+
+void Person::run() 
+{
+    
+}
```

Comment:
This `run` function is not identical to what was written in the video. 
Copy it exactly. 
then implement what the instructions are asking you.
If you are unsure what to do, ask a question in slack
- [ ] revise